src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -1027,6 +1025,13 @@ bool PolicyManagerImpl::HasCertificate() const {
   return !cache_->GetCertificate().empty();
 }
 
+void PolicyManagerImpl::SetDecryptedCertificate(
+    const std::string& certificate) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  cache_->SetDecryptedCertificate(certificate);
+  cache_->Backup();
+}
```

Comment:
Looks unused

Reply:
@jacobkeeler You are right. This function is specific to external proprietary policy flow. 
However this function should be overriden in both regular and external policy managers due to pure virtual function in interface class. So I will just remove function implementation for regular policy and  keep this function empty. See https://github.com/smartdevicelink/sdl_core/pull/2105/commits/8741ef28825dba9989c5b40924bc413a0c6c7383